client/cli/src/config.rs

```diff
@@ -477,6 +477,7 @@ pub trait CliConfiguration: Sized {
 			announce_block: self.announce_block()?,
 			role,
 			base_path: Some(base_path),
+			informant_output_format: sc_informant::OutputFormat::default(),
```

Comment:
That must go to a method that will provide the default by default like every other thing in this trait